Linux Script/configure-linux.sh

```diff
@@ -70,754 +70,835 @@ LOGGLY_USERNAME=
+        #          Syslog Logging Directives for Loggly ($LOGGLY_ACCOUNT.loggly.com)
+          #          -------------------------------------------------------
+        ##########################################################
+        ### RsyslogTemplate for Loggly ###
+          ##########################################################
+        \$template LogglyFormat,\"<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [$LOGGLY_AUTH_TOKEN@$LOGGLY_DISTRIBUTION_ID tag=\\\"RsyslogTLS\\\"] %msg%\n\"
+        # Setup disk assisted queues
+        \$WorkDirectory /var/spool/rsyslog # where to place spool files
+        \$ActionQueueFileName fwdRule1     # unique name prefix for spool files
+          \$ActionQueueMaxDiskSpace 1g       # 1gb space limit (use as much as possible)
+        \$ActionQueueSaveOnShutdown on     # save messages to disk on shutdown
+        \$ActionQueueType LinkedList       # run asynchronously
+        \$ActionResumeRetryCount -1        # infinite retries if host is down
+          #RsyslogGnuTLS
+        \$DefaultNetstreamDriverCAFile /etc/rsyslog.d/keys/ca.d/logs-01.loggly.com_sha12.crt
+        \$ActionSendStreamDriver gtls
+        \$ActionSendStreamDriverMode 1
+        \$ActionSendStreamDriverAuthMode x509/name
+        \$ActionSendStreamDriverPermittedPeer *.loggly.com
+        *.* @@$LOGS_01_HOST:$LOGGLY_SYSLOG_TLS_PORT;LogglyFormat
+        #################END CONFIG FILE#########################
+        "
+        inputStr_TLS_RSYS_8="
+        #          -------------------------------------------------------
+        #          Syslog Logging Directives for Loggly ($LOGGLY_ACCOUNT.loggly.com)
+          #          -------------------------------------------------------
+        # Setup disk assisted queues
+        \$WorkDirectory /var/spool/rsyslog # where to place spool files
+        \$ActionQueueFileName fwdRule1     # unique name prefix for spool files
+          \$ActionQueueMaxDiskSpace 1g       # 1gb space limit (use as much as possible)
+        \$ActionQueueSaveOnShutdown on     # save messages to disk on shutdown
+        \$ActionQueueType LinkedList       # run asynchronously
+        \$ActionResumeRetryCount -1        # infinite retries if host is down
+          #RsyslogGnuTLS
+        \$DefaultNetstreamDriverCAFile /etc/rsyslog.d/keys/ca.d/logs-01.loggly.com_sha12.crt
+        template(name=\"LogglyFormat\" type=\"string\"
+        string=\"<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [$LOGGLY_AUTH_TOKEN@$LOGGLY_DISTRIBUTION_ID tag=\\\"RsyslogTLS\\\"] %msg%\n\"
+        )
+        # Send messages to Loggly over TCP using the template.
+        action(type=\"omfwd\" protocol=\"tcp\" target=\"$LOGS_01_HOST\" port=\"$LOGGLY_SYSLOG_TLS_PORT\" template=\"LogglyFormat\" StreamDriver=\"gtls\" StreamDriverMode=\"1\" StreamDriverAuthMode=\"x509/name\" StreamDriverPermittedPeers=\"*.loggly.com\")
+        "
```

Comment:
These are the TLS configurations for Rsyslog version 7- and 8+. If user sets up for Non-TLS logging then [inputStr_NO_TLS](https://github.com/loggly/install-script/pull/85/files#diff-fcb43b5e6da2f59da41dc1549eb0fdf9R538) will be installed.